test/test.js

```diff
@@ -45,11 +45,11 @@ describe('vdom-streaming-serializer', function(){
 		count++;
 		if (count == 1) {
 			//assert.equal(1,2);
-			assert.equal(html, "<html><body><h1>Hello world</h1><ul>");
+			assert.equal(html, "<#document><html><body><h1>Hello world</h1><ul>");
```

Comment:
We actually don't want the `#document` part. That's not something you would send to the browser. What I would do is, inside of the `module.exports` function, check if the element is a document. Something like:

```js
if(element.nodeType === DOCUMENTTYPE) {
  element = element.firstChild;
}
```

https://developer.mozilla.org/en-US/docs/Web/API/Node/nodeType

That link gives you all of the different node types. Use the one for the #document instead of DOCUMENTTYPE